services/bots/src/github-webhook/handlers/docs_missing.ts

```diff
@@ -25,6 +25,8 @@ export class DocsMissing extends BaseWebhookHandler {
       PullRequestLabeledEvent | PullRequestUnlabeledEvent | PullRequestSynchronizeEvent
     >,
   ) {
+    const isReleasePR = context.payload.pull_request.base.ref === 'master';
```

Comment:
In an ideal world this is true, but it is possible to open a PR against master without it being a release.
I think this should have been `isPRAgainstMaster`